js/npm/sequelize/DataTypes.hx

```diff
@@ -28,14 +28,14 @@ class DataTypes {
 	}
 	/**
 	 * A 32 bit integer. 
-	 * Available properties: `length`, `unsigned:Bool`, `zerofill:Bool`
+	 * Available properties: `length:Int`, `unsigned:Bool`, `zerofill:Bool`
 	 */
 	static public inline function INTEGER(?options:{}):DataTypes {

```

Comment:
I use inline so i can return this wierd js code, for example: ```Sequelize['BIGINT'](options)```
Not sure i can invoke the same method using other js namespace or other haxe syntax.
This method works 100%.

Yes the require likely should be there to maintain consistency.

I'll check why there is no typedef for the first options.